test/jdk/java/net/httpclient/offline/FixedResponseHttpClient.java

```diff
@@ -40,19 +45,27 @@
 import java.net.http.HttpResponse.BodyHandler;
 import java.net.http.HttpResponse.BodySubscriber;
 import java.net.http.HttpResponse.PushPromiseHandler;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.nio.ByteBuffer.wrap;
 
 /**
  * An HttpClient that returns a given fixed response.
  * Suitable for testing where network connections are to be avoided.
+ * Can delegate to an actual HttpClient created from a supplied
+ * HttpClient.Builder if needed, by calling methods on its
+ * DelegatingClient super class.
```

Comment:
Minor typo, should have been `DelegatingHttpClient` instead of `DelegatingClient`